apps/sms/views/shared/js/bootstrap.js

```diff
@@ -1,5 +1,18 @@
-/* global Startup */
+/* global App,
+          Startup */
 
+(function() {
 'use strict';
 
+var shimHostIframe = document.querySelector('.shim-host');
```

Comment:
> I was thinking that this part could be in Startup.init but I'm fine if you want to sync with shim bootstrap)

Yeah, maybe it will be better to keep this in startup - will move it there.